android/vision/app/src/main/java/com/google/mediapipe/examples/objectdetection/MainActivity.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.mediapipe.examples.objectdetection
```

Comment:
Please refactor all com.google.mediapipe.examples.objectdetection paths to labgraph_vision.objectdetection paths.

Reply:
Done!